src/err.rs

```diff
@@ -394,6 +396,18 @@ impl <'p> std::convert::From<PythonObjectDowncastError<'p>> for PyErr {
     }
 }
 
+impl Error for PyErr {
+    fn description(&self) -> &str {
+        "Error originating from the rust-cpython bindings."
+    }
+}
+
+impl fmt::Display for PyErr {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "PyErr: ptype {:?} pvalue {:?}", self.ptype, self.pvalue)
```

Comment:
I would expect Display to result in something like `str(e)` in Python.
This code here is more like `repr(type(e))` + `repr(e)`, except even weirder if the exception is not yet initialized (`PyErr::new_lazy_init`).